applications/smart-distancing/eval/Dockerfile-eval

```diff
@@ -0,0 +1,17 @@
+# docker can be installed by following instructions: 
+# build: sudo docker build -f Dockerfile-amd64usbtpu-eval -t edgetpu-detector-eval .
+# run: sudo docker run -it -v /PATH_TO_CLONED_REPO_ROOT/:/repo detector-eval
+
+FROM python:3.6
+
+WORKDIR /repo/applications/smart-distancing/eval
+
+VOLUME /repo
+
+RUN pip install --upgrade pip setuptools==41.0.0 && pip install opencv-python==4.2.0.32 && pip install six==1.14.0 
+
+RUN pip install matplotlib==3.1.3
+
+ENTRYPOINT ["python", "pascal_evaluator.py"]
+
+CMD ["--gt", "eval_files/groundtruths/", "--det", "eval_files/detresults/", "-t", "0.5"]
```

Comment:
@mrn-mln can you please document these variables?

Reply:
I documented them at readme file.